pkg/mutation/system.go

```diff
@@ -139,6 +139,11 @@ func (s *System) Remove(id types.ID) error {
 	return nil
 }
 
+// Get mutator for given id
+func (s *System) Get(id types.ID) types.Mutator {
+	return s.mutatorsMap[id]
```

Comment:
Should we return a DeepCopy here do prevent any corruption of the cache caused by the caller?